blank-canvas-blocks/functions.php

```diff
@@ -12,6 +12,10 @@ function blank_canvas_blocks_support() {
 		// https://github.com/WordPress/gutenberg/issues/26901
 		add_theme_support( 'responsive-embeds' );
 
+		// Add support for Post Thumbnails on posts and pages.
+		// https://developer.wordpress.org/themes/functionality/featured-images-post-thumbnails/
+		add_theme_support( 'post-thumbnails' );
```

Comment:
without this the user can't add thumbnails to posts/pages, and those are kind of the point of this block pattern :D